vuepress/docs/v6.3.2/tutorials/customize-the-platform/pda-tutorial.md

```diff
@@ -1,52 +1,76 @@
+- The ent command line tool, installed and connected to your Kubernetes instance.
+
+### Installation Steps
+
+1. Apply the definitions for the bundle that comprises the PDA plugin. You'll need to adjust the `-n entando` option in each command to match your namespace or project.
+```
+ent bundler from-git -r https://github.com/entando-samples/entando-process-driven-plugin-bundle.git -d | ent kubectl apply -n entando -f -
+```
+
+2. Log into your App Builder instance.
+
+3. Select `Repository` from the menu on the left. Your bundles will be visible in the repository as shown in the screenshot below.
+
+4. Select `Install` to install the bundle. An installation can take several minutes while the application downloads the Linux images for the microservices and installs the related assets.
+
+5. Setup permissions to configure the service by following the [Initial Deploy instructions](https://github.com/entando/entando-process-driven-plugin#initial-deploy). Note that Step 1 in this sequence applies to Entando 6.3.0 only.
+
+6. Access the PDA plugn via one of the following options:
+
+**Option 1** If you'd like to make the PDA plugin your default home page, go to `App Builder → Pages → Settings`. In
+   the dropdown for Home Page, select `Home / xxxx` and click `Save`.
+
+You can now navigate to your plugin's home page using the home icon in the upper right of the App Builder.
+
+
+**Option 2** Alternatively, you can view the PDA plugn home page by going to `Pages → Management`, finding `xxxx` in the page tree, and clicking `View Published Page` from its actions.
+
+## Configuration
  
-For a discussion of components in the context of deployment, see [PDA Deployment Architecture](../../docs/concepts/pda-architecture.html#pda-mfes).
- 
-## Micro Frontend
- 
-A micro frontend architecture uses a composition of small frontend apps to develop a web application. Instead of writing
-a large monolith frontend application, the application is broken down
-into domain specific micro frontends, which are self-contained and can
-be developed and deployed independently.
- 
-Below is an overview of each micro frontend (MFE) that is available as part of the Entando PDA plugin. Details specific to the PAM implementation of an MFE are provided where appropriate. MFE behavior and datasources must be defined if the integration layer is extended to other
-engines or custom implementations.
+The following discussion guides the configuration of PDA features and capabilities while introducing the plugin building blocks.
  
+For a discussion of components in the context of deployment, see [PDA Deployment Architecture](../../docs/concepts/pda-architecture.html#pda-mfes).
+
+## Application Details
+
+The Entando Standard Banking Demo application demonstrates a number of the major features of the Entando platform, including
+
+- Micro Frontends
+- Process Automation Manager
```

Comment:
should have (PAM) here so you can use it below. Unless I missed it.

Reply:
i took out the acronyms here :P just because i wanted to keep the full names in the paragraphs first introducing the concepts/terms

Reply:
i provide acronyms below

Reply:
But the first time 'PAM' appears is where the full name and abbreviation needs to appear.

Reply:
cry. ok will change. i was hoping to get away without acronyms cluttering the bullets.